homeassistant/components/google_drive/sensor.py

```diff
@@ -0,0 +1,116 @@
+"""Support for GoogleDrive sensors."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import EntityCategory, UnitOfInformation
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.util import slugify
+
+from .coordinator import (
+    GoogleDriveConfigEntry,
+    GoogleDriveCoordinatorData,
+    GoogleDriveDataUpdateCoordinator,
+)
+from .entity import GoogleDriveEntity
+
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
+
+
+@dataclass(frozen=True, kw_only=True)
+class GoogleDriveSensorEntityDescription(SensorEntityDescription):
+    """Describes GoogleDrive sensor entity."""
+
+    value_fn: Callable[[GoogleDriveCoordinatorData], datetime | StateType]
```

Comment:
I don't see any sensor that has datetime

Reply:
Fixed. Copy paste mistake.